libafl_frida/src/script.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
```

Comment:
Maybe move this into the fuzzer example (or even create a new fuzzer which uses this functionality).

Reply:
This relates to the above. The idea is to add a namespace and API to the JS layer which the user can call to interact with `libafl_frida` itself. e.g. the functions exported would not be target specific.